pkg/executor/sortexec/parallel_sort_spill_helper.go

```diff
@@ -124,18 +124,25 @@ func (p *parallelSortSpillHelper) spill() (err error) {
 	workerWaiter := &sync.WaitGroup{}
 	workerWaiter.Add(workerNum)
 	sortedRowsIters := make([]*chunk.Iterator4Slice, workerNum)
+	errChannel := make(chan error, workerNum*2)
```

Comment:
Why workerNum*2?

Reply:
> Why workerNum*2?

One worker may put to err into channel respectively at L145 and L132.